addons/api/mirage/factories/session.js

```diff
@@ -15,7 +16,8 @@ export const pickRandomStatusString = () =>
 
 export default factory.extend({
   id: () => generateId('ss_'),
-
+  authorized_actions: () =>
+    permissions.authorizedActionsFor('session') || ['read', 'cancel'],
```

Comment:
not a blocker- should we also include read:self and cancel:self here?

Reply:
That's somewhat redundant since read covers `read:self` and same with cancel